runbot_website_display/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```

Comment:
Put license link like in https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst